src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIBookingService.kt

```diff
@@ -14,7 +14,7 @@ import javax.validation.constraints.NotNull
 @Service
 @Transactional
 class CommunityAPIBookingService(
-  @Value("\${appointments.bookings.enabled}") private val bookingsEnabled: Boolean,
+  @Value("\${community-api.appointments.bookings.enabled}") private val bookingsEnabled: Boolean,
```

Comment:
also has mention in helm values

Reply:
thanks! value removed from helm since it was set to true (the default) anyway